src/actions/walletActions.js

```diff
@@ -160,6 +153,8 @@ export const generateWalletMnemonicAction = (mnemonicPhrase?: string) => {
         wordsToValidate,
       },
     });
+
+    Answers.logCustom('Create Wallet');
```

Comment:
Hm... In the list above I see that we should track "Wallet created" event.
On this step app just generates mnemonic. But wallet does not get registered anywhere yet.
Maybe to track wallets that have been created (not intended to be created) `getTokenWalletAndRegister` (could be found in `onboardingActions.js`) would be more accurate.

Reply:
Yeah, that's a better option, thanks! Updated.